packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,164 @@
+   * @param account Address of the account being mapped to the identifier
+   * @param issuedOn Time at which the issuer issued the attestation in Unix time
+   * @param signer Address of the signer of the attestation
+   * @param v The recovery id of the incoming ECDSA signature
+   * @param r Output value r of the ECDSA signature
+   * @param s Output value s of the ECDSA signature
+   * @dev Throws if an attestation with the same (identifier, issuer, account) already exists
+   */
+  registerAttestation: (
+    identifier: string | number[],
+    issuer: string,
+    account: string,
+    signer: string,
+    issuedOn: number | string,
+    v: number | string,
+    r: string | number[],
+    s: string | number[]
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.registerAttestation)
+
+  /**
+   * @notice Revokes an attestation
+   * @param identifier Hash of the identifier to be revoked
+   * @param issuer Address of the attestation issuer
+   * @param account Address of the account mapped to the identifier
+   * @dev Throws if sender is not the issuer, signer, or account
+   */
+  revokedAttestations: (
+    identifier: string | number[],
+    issuer: string,
+    account: string
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.revokeAttestation)
+
+  /**
+   * @notice Revokes attestations [identifiers <-> accounts] from issuer
+   * @param issuer Address of the issuer of all attestations to be revoked
+   * @param identifiers Hash of the identifiers
+   * @param accounts Addresses of the accounts mapped to the identifiers
+   *   at the same indices
+   * @dev Throws if the number of identifiers and accounts is not the same
+   * @dev Throws if sender is not the issuer or currently registered signer of issuer
+   * @dev Throws if an attestation is not found for identifiers[i] <-> accounts[i]
+   */
+  batchRevokeAttestations: (
+    issuer: string,
+    identifiers: (string | number[])[],
+    accounts: string[]
+  ) => Promise<void> = proxySend(this.connection, this.contract.methods.batchRevokeAttestations)
+
+  /**
+   * @notice Validates the given attestation and signature
+   * @param identifier Hash of the identifier to be attested
+   * @param issuer Address of the attestation issuer
+   * @param account Address of the account being mapped to the identifier
+   * @param issuedOn Time at which the issuer issued the attestation in Unix time
+   * @param signer Address of the signer of the attestation
+   * @param v The recovery id of the incoming ECDSA signature
+   * @param r Output value r of the ECDSA signature
+   * @param s Output value s of the ECDSA signature
+   * @dev Throws if attestation has been revoked
+   * @dev Throws if signer is not an authorized AttestationSigner of the issuer
+   */
+  validateAttestationSig: (
+    identifier: string | number[],
+    issuer: string,
+    account: string,
+    signer: string,
+    issuedOn: number | string,
+    v: number | string,
```

Comment:
super nitty but it might be helpful to keep the same order `string | number` for all that are that type

Reply:
I agree - did this local to the function